polaris-core/src/main/java/org/apache/polaris/core/entity/CatalogEntity.java

```diff
@@ -190,6 +202,21 @@ public Catalog.TypeEnum getCatalogType() {
         .orElse(null);
   }
 
+  public boolean isPassthroughFacade() {
```

Comment:
It seems like we are piggybacking on the existing EXTERNAL concept even though the semantics are very different. For example, it'll be invalid for a passthrough catalog to receive a notification right?

Reply:
It's actually intentional for a federated catalog to directly extend the EXTERNAL concept. It is allowed for a passthrough catalog to receive a notification to facilitate any JIT-creation or caching semantics, as long as the catalog implementation is an instanceof `SupportsNotifications`.